usr/src/cmd/zdb/zdb.c

```diff
@@ -1496,7 +1497,7 @@ dump_full_bpobj(bpobj_t *bpo, char *name, int indent)
 		    (u_longlong_t)bpo->bpo_phys->bpo_subobjs,
 		    bytes, comp, uncomp);
 
-		for (uint64_t i = 0; i < bpo->bpo_phys->bpo_num_subobjs; i++) {
+		for (i = 0; i < bpo->bpo_phys->bpo_num_subobjs; i++) {
```

Comment:
This change seems like a readability regression.  As to consistency with the rest of this file, there are several other instances of this style of declaration, dating back to at least 2008.  Other similar code should join this millennium and adopt c99-style declarations where they improve readability.